mergin/client.py

```diff
@@ -700,41 +718,67 @@ def project_info(self, project_path_or_id, since=None, version=None):
             resp = self.get("/v1/project/{}".format(project_path_or_id), params)
         return json.load(resp)
 
-    def project_versions(self, project_path, since=None, to=None):
+    def project_versions_page(self, project_path, page, per_page=100, descending=False):
+        params = {"page": page, "per_page": per_page, "descending": descending}
+        resp = self.get("/v1/project/versions/paginated/{}".format(project_path), params)
+        resp_json = json.load(resp)
+        return resp_json["versions"]
+
+    def project_versions_count(self, project_path):
+        """
+        return the total count of versions
+
+        To note the requested informations are kept to the minimal
```

Comment:
> To note the requested informations are kept to the minimal
Not sure what this means! :grin:

Reply:
Clarified comment and use docstring format